src/sentry/backup/comparators.py

```diff
@@ -865,6 +865,7 @@ def get_default_comparators() -> dict[str, list[JSONScrubbingComparator]]:
             ],
             "sentry.userrole": [DateUpdatedComparator("date_updated")],
             "sentry.userroleuser": [DateUpdatedComparator("date_updated")],
+            "workflow_engine.datacondition": [DateUpdatedComparator("date_updated", "date_added")],
```

Comment:
You'll need to add all the new models here

Reply:
👍 thanks - yeah, i was avoiding doing all those changes initially just cause of all the things. updated! (hopefully the tests are green now too 🤞)